src/main/java/platform/codingnomads/co/springweb/resttemplate/POST/postForObject/Models/UserTemplate.java

```diff
@@ -0,0 +1,15 @@
+package platform.codingnomads.co.springweb.resttemplate.POST.postForObject.Models;
+
+import lombok.Builder;
+import lombok.Data;
+
+@Data
+@Builder
+public class UserTemplate {
+    private long id;
+    private String email;
+    private String first_name;
+    private String last_name;
+    private long createdAt;
+    private long updatedAt;
```

Comment:
Same note about primitives. 
In this case, the UserTemplate will have `id`, 'createdAt' and 'updatedAt' would be `0` when you create the object.
Since you use the template to PATCH the resulting JSON will have the parameters with`0` values (unless you `set` them) and so Server may recognize the values as "new values to be PATCHed" and you may lose data, or "valid values for POST" and you will have `0` in DB that may be wrong. 

In this specific case, all `id`, 'createdAt' and 'updatedAt' are special properties that are maintained by SQL, so it will not change anything on the server.

Reply:
Ok, and I'm realizing now that most of the examples I've been seeing in Coding Nomads have use the Long type instead of long, for example, so I guess that's the better way to go.